packages/augur-core/tests/conftest.py

```diff
@@ -404,95 +432,101 @@ def uploadAugur(self):
         # We have to upload Augur first
         return self.upload("../source/contracts/Augur.sol")
 
-    def uploadShareToken(self, augurAddress = None):
-        augurAddress = augurAddress if augurAddress else self.contracts['Augur'].address
-        self.ensureShareTokenDependencies()
-        shareTokenFactory = self.contracts['ShareTokenFactory']
-        shareToken = shareTokenFactory.createShareToken(augurAddress)
-        return self.applySignature('shareToken', shareToken)
-
     def createUniverse(self):
-        universeAddress = self.contracts['Augur'].createGenesisUniverse()
+        augur = self.contracts['Augur']
+        assert augur.createGenesisUniverse(getReturnData=False)
+        universeCreatedLogs = augur.getLogs("UniverseCreated")
+        universeAddress = universeCreatedLogs[0].args.childUniverse
         universe = self.applySignature('Universe', universeAddress)
         assert universe.getTypeName() == stringToBytes('Universe')
         return universe
 
-    def getShareToken(self, market, outcome):
-        shareTokenAddress = market.getShareToken(outcome)
-        assert shareTokenAddress
-        shareToken = ABIContract(self.chain, ContractTranslator(ContractsFixture.signatures['ShareToken']), shareTokenAddress)
-        return shareToken
-
-    def getOrCreateChildUniverse(self, parentUniverse, market, payoutDistribution):
-        assert payoutDistributionHash
-        childUniverseAddress = parentUniverse.getOrCreateChildUniverse(payoutDistribution)
-        assert childUniverseAddress
-        childUniverse = ABIContract(self.chain, ContractTranslator(ContractsFixture.signatures['Universe']), childUniverseAddress)
-        return childUniverse
-
-    def createYesNoMarket(self, universe, endTime, feePerCashInAttoCash, affiliateFeeDivisor, designatedReporterAddress, sender=tester.k0, topic="", extraInfo="{description: '\"description\"}", validityBond=0):
-        marketCreationFee = validityBond or universe.getOrCacheMarketCreationCost()
+    def distributeRep(self, universe):
+        # Get the reputation token for this universe and migrate legacy REP to it
+        reputationToken = self.applySignature('ReputationToken', universe.getReputationToken())
+        legacyRepToken = self.applySignature('LegacyReputationToken', reputationToken.getLegacyRepToken())
+        totalSupply = legacyRepToken.balanceOf(self.accounts[0])
+        legacyRepToken.approve(reputationToken.address, totalSupply)
+        reputationToken.migrateFromLegacyReputationToken()
+
+    def getLogValue(self, eventName, argName, last=False):
```

Comment:
This should probably be two functions: getFirstLogValue and getLastLogValue.

Reply:
Never used so I'm gonna just remove it